gnomad_qc/v5/resources/basics.py

```diff
@@ -103,36 +104,36 @@ def get_logging_path(
 
 
 def get_aou_vds(
-    test: bool = False,
     split: bool = False,
+    filter_samples: Optional[Union[List[str], hl.Table]] = None,
+    test: bool = False,
+    filter_partitions: Optional[List[int]] = None,
     chrom: Optional[Union[str, List[str], Set[str]]] = None,
     autosomes_only: bool = False,
     sex_chr_only: bool = False,
-    filter_samples: Optional[Union[List[str], hl.Table]] = None,
-    filter_partitions: Optional[List[int]] = None,
+    filter_variant_ht: Optional[hl.Table] = None,
     filter_intervals: Optional[List[Union[str, hl.tinterval]]] = None,
     split_reference_blocks: bool = True,
     remove_dead_alleles: bool = True,
-    filter_variant_ht: Optional[hl.Table] = None,
     entries_to_keep: Optional[List[str]] = None,
     checkpoint_variant_data: bool = False,
     naive_coalesce_partitions: Optional[int] = None,
 ) -> hl.vds.VariantDataset:
     """
     Load the AOU VDS.
 
-    :param test: Whether to load the test VDS. The test VDS includes 10 samples selected from the full dataset for testing purposes. Default is False.
     :param split: Whether to split the multi-allelic variants in the VDS. Note: this will perform a split on the VDS
         rather than grab an already split VDS. Default is False.
+    :param filter_samples: Optional samples to filter the VDS to. Can be a list of sample IDs or a Table with sample IDs.
+    :param test: Whether to load the test VDS. The test VDS includes 10 samples selected from the full dataset for testing purposes. Default is False.
```

Comment:
```suggestion
    :param test: Whether to load the test VDS instead of the full VDS. The test VDS includes 10 samples selected from the full dataset for testing purposes. Default is False.
```